Android/app/src/main/java/io/github/project_travel_mate/MyFriendsFragment.java

```diff
@@ -113,7 +113,7 @@ public void onResponse(Call call, final Response response) throws IOException {
                                 return;
                             }
 
-                            for (int i = 0; i < arr.length() - 1; i++) {
+                            for (int i = 1; i < arr.length(); i++) {
```

Comment:
Make it start from 0 only. I have created an issue in the server to don't include friend's own info(https://github.com/project-travel-mate/server/issues/100)